packages/webapp/pages/onboarding.tsx

```diff
@@ -48,13 +50,20 @@ const versionToTitle: Record<OnboardingFilteringTitle, string> = {
   [OnboardingFilteringTitle.V4]: 'Choose the topics you’re passionate about',
 };
 
-const Title = classed('h2', 'font-bold typo-title2');
+const Title = classed('h2', 'font-bold typo-title1');
 
 const maxAuthWidth = 'tablet:max-w-[30rem]';
 
+// Because of the video colours, we need to adjust the background colour a little bit
+// eslint-disable-next-line @dailydotdev/daily-dev-eslint-rules/no-custom-color
 const Container = classed(
   'div',
-  'flex flex-col overflow-x-hidden items-center min-h-[100vh] w-full h-full max-h-[100vh] flex-1 z-max bg-theme-bg-primary',
+  'flex flex-col overflow-x-hidden items-center min-h-[100vh] w-full h-full max-h-[100vh] flex-1 z-max bg-[#0A1119]',
+);
+
+const OnboardingTitle = classed(
+  'h1',
+  'mb-4 font-bold text-transparent bg-clip-text bg-gradient-to-r from-theme-color-bacon to-theme-color-cabbage',
 );
 
 const seo: NextSeoProps = {
```

Comment:
In general this page is getting very bloated, maybe we should consider moving some stuff to smaller components to break it down a bit better?